Xamarin.Forms.Platform.UAP/CollectionView/ObservableItemTemplateCollection.cs

```diff
@@ -56,28 +60,38 @@ public void CleanUp()
 
 		void InnerCollectionChanged(object sender, NotifyCollectionChangedEventArgs args)
 		{
-			switch (args.Action)
+			if (SynchronizationContext.Current != _synchronizationContext)
+				_synchronizationContext.Post(RaiseCollectionChanged, args);
```

Comment:
```suggestion
			{
				_synchronizationContext.Post(RaiseCollectionChanged, args);
			}
```

Maybe it's my personal preference, but can we have brackets in here? Seems a bit more readable to me with the `else` in there. Also, more in line with the other code that has this change